src/pydvl/value/oob/oob.py

```diff
@@ -30,8 +27,8 @@ def compute_data_oob(
     *,
     n_est: int = 10,
     max_samples: float = 0.8,
-    loss: Callable = None,
-    n_jobs: int = None,
+    n_jobs: Optional[int] = None,
+    loss: Optional[LossFunction] = None,
```

Comment:
why this permutation? n_jobs is usually among the last parameters.

Reply:
Must have been accidental. I will permute them back.